src/main/scala/goatrodeo/omnibor/BuildGraph.scala

```diff
@@ -166,35 +166,34 @@ object BuildGraph {
           (fileType.sourceGitOid() match {
             case None => TreeSet[Edge]()
             case Some(source) =>
-              TreeSet[Edge]((EdgeType.BuiltFrom, source))
+              TreeSet[Edge]((EdgeType.builtFrom, source))
           })
           ++
           // include parent back-reference
           (parent match {
             case Some(parentId) =>
-              Vector[Edge]((EdgeType.ContainedBy, parentId))
+              Vector[Edge]((EdgeType.containedBy, parentId))
             case None => topConnections
           })
           ++
           // include aliases only if we aren't merging this item (if we're)
           // merging, then the aliases already exist and no point in regenerating them
-          (aliases.map(alias => (EdgeType.AliasFrom, alias))).toSet
+          (aliases.map(alias => (EdgeType.aliasFrom, alias))).toSet
 
 
         val item = Item(
           identifier = mainFileGitOID,
           reference = Item.noopLocationReference,
           connections = computedConnections,
-          
-          metadata = Some(
+          bodyMimeType = Some("application/goatrodeo"),
```

Comment:
Should use a vnd mime type

Reply:
Can you tell me what the string should be?

Reply:
`application/vnd.spice-labs.goatrodeo` makes sense.

Reply:
Or even `application/vnd.io.spice-labs.goatrodeo`

Reply:
I think we'd omit the dash in the `.io` version.

Reply:
I set it to `application/vnd.cc.goatrodeo` as https://goatrodeo.cc points to the open source project.

Reply:
Oh perfect!